Jellyfin.Networking/Manager/NetworkManager.cs

```diff
@@ -892,31 +940,34 @@ private bool MatchesPublishedServerUrl(IPAddress source, bool isInExternalSubnet
             bindPreference = string.Empty;
             int? port = null;
 
-            var validPublishedServerUrls = _publishedServerUrls.Where(x => x.Key.Address.Equals(IPAddress.Any)
-                                                || x.Key.Address.Equals(IPAddress.IPv6Any)
-                                                || x.Key.Subnet.Contains(source))
-                                            .DistinctBy(x => x.Key)
-                                            .OrderBy(x => x.Key.Address.Equals(IPAddress.Any)
-                                                || x.Key.Address.Equals(IPAddress.IPv6Any))
+            // Only consider subnets including the source IP, prefering specific overrides
+            var validPublishedServerUrls = new List<PublishedServerUriOverride>();
```

Comment:
isn't this empty list unused?

Reply:
the list seems to be filled-in in the if-else below and the used in the foreach afterwards :)

Reply:
It is not filled in, it's re-assigned